man/systemd.journal-fields.xml

```diff
@@ -56,10 +56,10 @@
     their syntax but with fields that can include binary data.
     Primarily, fields are formatted UTF-8 text strings, and binary
     formatting is used only where formatting as UTF-8 text strings
-    makes little sense. New fields may freely be defined by
-    applications, but a few fields have special meaning. All fields
-    with special meanings are optional. In some cases, fields may
-    appear more than once per entry.</para>
+    makes little sense. Applications can add custom fields, and
+    those fields can occur more than once. systemd-journald will 
+    attach special fields (starting with an underscore). 
```

Comment:
The underscore fields can also appear multiple times, for example _UDEV_DEVLINK=, where we even document that explicitly

Reply:
Maybe it makes sense to break it down for each field?

Reply:
Well, that's kinda what already happens: we do document that that's a field that may show up multiple times.

Note though, that existing fields might show up multiple times in the same record in the future too. For example when we encounter a dependency loop we'll probably log in future a record that contains _SYSTEMD_UNIT= for all units involved in the loop, so that you can use any of them as key to find information about the loop. (There's at least a TODO list item about this.) It's one of the reasons this has been documented a bit vaguely so far: things might change sooner or later...

Reply:
This wouldn't make much sense... but I guess that with you would match on both when searching, and show either pseudorandomly when displaying.